frontend/package.json

```diff
@@ -38,7 +38,8 @@
     "dev": "next dev",
     "export": "next build && next export -o build",
     "ts-reignore": "ts-migrate reignore . --sources='src/**/*'",
-    "ts-validate": "tsc -p tsconfig.json"
+    "ts-validate": "tsc -p tsconfig.json",
+    "prepare": "bash ./git-secrets-conf.sh"
```

Comment:
suggestion(non-blocking): I would suggest a different name for this npm script, something that tells more about what it is, since "prepare" sounds to me a little to general.
Even something like "setup-git-secrets" would be fine, or something like that so that no doubts can arise.

Reply:
The "prepare" is a part of npm lifecycle see https://docs.npmjs.com/cli/v8/using-npm/scripts. This is to ensure the script it's executed at every npm install. So I cannot change that name.

Reply:
Perfect then! Thanks!